### ISSUE TITLE
artifacts around bright stars

### ISSUE THREAD

**Issue by esheldon** [LSSTDESC/imSim#481]
I'm seeing artifacts around bright stars.   I've attached an image showing an example.  You can see what might be FFT artifacts as vertical/horizontal features.   there are no detector effects here.

I can provide an instcat and yaml file if needed

![artifact01](https://github.com/user-attachments/assets/98bb4738-014e-4c50-872f-cfc54d37b04d)

**Comment by cwwalter** [comment]
Well... that looks like the airy rings with the modulation on top of it..

Is is a saturated object? Is so, see this discussion about how we assume in this case it is monochromatic so they don't get washed out.

https://lsstc.slack.com/archives/CHXKSF3HC/p1695182534709269?thread_ts=1695175590.496979&cid=CHXKSF3HC

 I don't know if I have seen then just in a line like that though.

**Comment by jmeyers314** [comment]
Hmm..  I think if this were Airy the rings would circle all the way around.  FFT artifact feels right to me.

**Comment by rmjarvis** [comment]
Yeah, that's the kind of FFT artifact that typically happens in bright stars.  You need pretty huge FFTs to avoid them (via smaller than normal folding_threshold and maxk_threshold).

**Comment by esheldon** [comment]
![detected](https://github.com/user-attachments/assets/53c2398a-6add-451a-86a1-1583239851e1)

**Comment by esheldon** [comment]
Example in the amplifier image with detector effects
![amp-artifacts](https://github.com/user-attachments/assets/4ca0bea8-aa51-41bb-88bd-1844a5972c93)

**Comment by esheldon** [comment]
Here is a tar ball with config file, instcat, and a script showing how to run it

https://www.cosmo.bnl.gov/www/esheldon/data/lsst/test-artifacts.tar
